src/styles/StyleUtils.js

```diff
@@ -699,15 +701,15 @@ function getReportWelcomeBackgroundImageStyle(isSmallScreenWidth) {
  * @param {Boolean} isSmallScreenWidth
  * @returns {Object}
  */
-function getReportWelcomeBackgroundImageViewStyle(isSmallScreenWidth) {
+function getReportWelcomeTopMarginStyle(isSmallScreenWidth) {
```

Comment:
You'll need to remove/update the description for this methods's docblock since it's not relevant anymore :)